### PR TITLE
Commit processing: handle for event “Device Was Disconnect”

### PR DESCRIPTION
**Pull request by alex-moc**

### CODE REVIEW

**Comment by alex-moc** [comment]
May be we can postpone tests for this functionality to next release? I high loaded now.

**Comment by Clark73** [comment]
Personally, I wont sign off on this PR without tests because Im scared of what is being added here could break the existing code and I want that reassurance. But if you can convince @ssteveli that they are not needed then Ill drop it.

**Comment by ssteveli** [comment]
Agreed, we need test coverage... there has been to much instability reported on the SDK from our customers.

File: FitpaySDK/Rtm/RtmMessageHandlers/RtmMessageHandlerV2.swift [hunk 1]
```diff
@@ -72,21 +72,27 @@ class RtmMessageHandlerV2: NSObject, RtmMessageHandler {
...
-                                                             type: RtmMessageTypeVer2.sync.rawValue,
-                                                             success: false), retries: 3)
-                delegate.show(status: .syncError, message: "Can't make sync. Session data or user is nil.", error: nil)
-            }
+        
+        guard let _ = self.webViewSessionData,
+            let user = self.wvConfigStorage.user,
+            let deviceInfo = self.wvConfigStorage.device,
+            let paymentDevice = self.wvConfigStorage.paymentDevice else {
+                log.warning("WV_DATA: rtm not yet configured to hand syncs requests, failing sync.")
```
Comment (ssteveli) [inline FitpaySDK/Rtm/RtmMessageHandlers/RtmMessageHandlerV2.swift]: `hand` should be `handle`

**Comment by walkline** [comment]
Added some unit tests, details in description of 753ee5a114a476b9f59543b3d6c18f77cfc75543 commit.
Also, I have a question. Is it normal behavior when we are failing sync if payment device was disconnected?

**Pull request merged**